### DISCUSSION TITLE
Wires soldering

### DISCUSSION THREAD

**Discussion by MartinSaaby** [Louisvdw/dbus-serialbattery#180] [Q&A] [answered]
Hi 

I have a Victron system with a running Venus os on a rpi.
For bms I have a Daly 250A smart bms.

I need some help connecting the wires :)

My plan is to use the RS485/can port with the included wires.
I have postet some pictures of the two cables. Do anyone know the right color/text combination ?

**Victron RS485 -> USB**
![IMG_2334](https://user-images.githubusercontent.com/88431885/184580007-2de0c388-2dc9-4775-b417-8201025a46fc.jpeg)
 
**Cable included with Daly BMS**
![IMG_2333](https://user-images.githubusercontent.com/88431885/184580181-771d1227-3d7d-490a-8770-a2dfaab073b9.jpeg)

**Comment by pau1phi11ips** [comment, marked as answer]
See here: https://www.victronenergy.com/media/pg/Energy_Meter_ET112/en/installation-and-configuration.html

Data+ / A: Orange
Data- / B: Yellow
Ground: Black

**Comment by MartinSaaby** [comment]
Thanks a lot! :)
